### DISCUSSION TITLE
Welcome to livewire-csv Discussions!

### DISCUSSION THREAD

**Discussion by askdkc** [askdkc/livewire-csv#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about. / なんでも聞いてね？
  * Share ideas. / アイデアがあればここに（出来るとは言っていない）
  * Engage with other community members. / みんな仲良く
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by valeriomatranga** [comment]
Hi, I'm using this package.
I would like to thank you for your work and recommend adding an emit to data entry that will allow you to get the records you just entered.
![Code_DXCjysfOZx](https://github.com/askdkc/livewire-csv/assets/78118886/aaf24c46-0f99-4833-ab7c-1ef4a251b87f)

**Reply by askdkc** [reply]
@valeriomatranga 

interesting! Could you make a PR?

**Reply by algomera** [reply]
hi, I created the pull request as requested

**Reply by askdkc** [reply]
@algomera 

Thanks. Merged and released!

https://github.com/askdkc/livewire-csv/releases/tag/v1.6.2

**Comment by algomera** [comment]
hi, I would propose to insert a variable to set the column delimiter
![Code_wTOuLCnm5T](https://github.com/askdkc/livewire-csv/assets/107032524/f61684a2-83e6-4842-98ae-7f83947828d3)
![Code_MLmjJE12oe](https://github.com/askdkc/livewire-csv/assets/107032524/f8d67065-3b6c-4146-b31d-75fbc1e3faf4)

**Reply by askdkc** [reply]
Thank you for the PR.
Could you please write a test for this? 🙏
